api/src/main/java/org/apache/cloudstack/api/response/UserVmResponse.java

```diff
@@ -269,6 +270,10 @@ public class UserVmResponse extends BaseResponseWithTagInformation implements Co
     @Param(description = "the hypervisor on which the template runs")
     private String hypervisor;
 
+    @SerializedName(ApiConstants.IP_ADDRESS)
+    @Param(description = "the VM's primary IP address")
+    private String ipAddress;
```

Comment:
Is there a way to avoid this, the ip address table can cause excessive scans in the DB.

Reply:
I don't see a reason to only return the primary IP on the metrics response, I think it should be returned when listing the user VMs as well. Furthermore, the metrics response extends the UserVmResponse, so when listing the metrics the IP will still be returned.